src/sentry/tagstore/snuba/backend.py

```diff
@@ -749,8 +749,17 @@ def delete_all_group_tag_values(self, project_id, group_id):
 
     def delete_tag_key(self, project_id, key):
         # Called by ``ProjectTagKeyDetailsEndpoint.delete``. The return value
-        # is not used.
-        pass
+        # is used for audit logging.
+        try:
+            return [
+                self.get_tag_key(
```

Comment:
This doesn't technically need to do the work of fetching the tag key, but I figured it's better to return an empty result if the tag deletion isn't going to actually do anything. If anyone has strong opinions that we *shouldn't* run this (potentially unnecessary) query, I can just return a synthesized `TagKey` record here instead.

Reply:
Deletes happen so rarely that I'm all for being correct.